spec/factories/users.rb

```diff
@@ -1,14 +1,13 @@
-require 'faker'
+require "faker"
 
 FactoryGirl.define do
   factory :user do
-    first_name             { Faker::Name.first_name }
-    last_name              { Faker::Name.last_name }
-    position               'Advisor'
-    organization           { Faker::Company.name }
-    email                  { Faker::Internet.email("#{last_name}") }
-    password               { Faker::Internet.password }
-
+    first_name         { Faker::Name.first_name }
+    last_name          { Faker::Name.last_name }
+    position           "Advisor"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.